index.html

```diff
@@ -135,6 +135,72 @@ <h3><a href="#command-bootstrap">
           </p>
         </section>
 
+        <section class="command" id="command-init">
+          <h3><a href="#command-init">
+            <code>lerna create &lt;package name&gt; &lt;location&gt;</code>
+          </a></h3>
+
+          <p>
+            Create a new lerna-managed package.
+          </p>
+          <p>
+            <strong>
+                The package name (including scope), which must be locally unique and
+                publicly available.
+            </strong>
+          </p>
+          <p>
+            <strong>
+              Location will default to the first configured package
+              location.
+            </strong>                   
+          </p>
+        </section>
+
+        <section class="command" id="command-ls">
+          <h3><a href="#command-ls">
+            <code>lerna add &lt;package&gt;</code>
+          </a></h3>
+
+          <p>
+            Add a dependency to matched packages.
+          </p>
+          <p>
+            <p>
+              <code>
+                # Install module-1 to module-2 <br/>
```

Comment:
Add module-1 as a dependency of module-2 (install is one-time, adding as a dependency is permanent)